ocdev/plugins/setup/setup.py

```diff
@@ -166,6 +166,12 @@ def clone_repo(self, url, branch='master', directory=None, no_history=False):
 
         if directory:
             cmd.append(directory)
-
-
-        return check_call(cmd)
\ No newline at end of file
+        assert os.getcwdu() == '/var/www', "current working directory is {0}, needs to be '/var/www'".format(os.getcwdu())
```

Comment:
This is platform dependent. For development I usually clone it into a folder in my home directory